content/applications/inventory_and_mrp/inventory/shipping_receiving/advanced_operations_shipping/print_on_validation.rst

```diff
@@ -0,0 +1,156 @@
+=================================
+Print delivery slip on validation
+=================================
+
+Automatically print delivery slips that contain details about a package's recipient, contents, or
+handling instructions.
+
+In Odoo, the PDFs of the following documents can be configured to print upon validating an
+*Inventory* operation (e.g. receipt, picking, delivery orders, quality checks):
+
+#. Delivery slip
+#. Return slip
+#. Product labels of items in the order
+#. Lot and serial number labels
+#. Carrier labels
+#. Export documents
+#. Package Content
+#. Package label
+
+.. _inventory/shipping_receiving/print_setup:
+
+To automatically print these forms, navigate to :menuselection:`Inventory app --> Configuration -->
+Operations Types` and select the desired operation type.
+
+In the :guilabel:`Hardware` tab, tick each of the options in the :guilabel:`Print on Validation`
```

Comment:
```suggestion
In the :guilabel:`Hardware` tab, tick each of the desired options available in the :guilabel:`Print on Validation`
```

Reply:
Can the user select just one of these options, or do all of them have to be selected?